algorithms/scaling/model/model.py

```diff
@@ -122,15 +122,35 @@
     .type = bool
     .help = "Option to turn off absorption correction (default True if oscillation > 60.0)."
     .expert_level = 1
-lmax = 4
+absorption_level = low medium high
+    .type = choice
+    .help = "Expected degree of relative absorption for different scattering"
+            "paths through the crystal(s). If an option is selected, the"
+            "scaling model parameters lmax and surface_weight will be set to"
+            "appropriate values."
+            "Relative absorption increases as crystal size increases,"
+            "increases as wavelength increases and is increased as the crystal"
+            "dimensions become less equal (i.e. is higher for needle shaped"
+            "crystals and zero for a spherical crystal)."
+            "Definitions of the levels and approximate correction magnitude:"
+            "low:    ~1%% relative absorption, expected for crystals on the order"
```

Comment:
protein crystals - 100 µm germanate (say) will absorb a lot more.

Reply:
perhaps 'typical protein crystal' would be a better description i.e. containing only CHNOS?